src/plugins/discover/public/application/main/services/discover_state.ts

```diff
@@ -165,7 +150,7 @@ export interface DiscoverStateContainer {
   };
 }
 
-const APP_STATE_URL_KEY = '_a';
+export const APP_STATE_URL_KEY = '_a';
```

Comment:
I thought I left a comment on this in my last review, but I can't find it anywhere, so maybe I'm just imaging things 😅. It seems like this is only used in `discover_app_state_container` and nowhere else, so it might make sense to move it there instead of exporting from here.

Reply:
sure thing! done